extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -725,11 +726,11 @@ private function loadUserBadges() {
 	public static function canEarnBadges( User $user = null ) {
 		global $wgWikiaBotLikeUsers, $wgUser, $wgAchievementsExemptUsers;
 
-		if ( empty( $user ) ) {
+		if( empty( $user ) ) {
 			$user = $wgUser;
 		}
 
-		if (
+		if(
 			$user->isAnon() ||
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (7) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)